kapitan/targets.py

```diff
@@ -269,7 +269,8 @@ def save_inv_cache(compile_path, targets):
 
 
 def load_target_inventory(inventory_path, targets):
-    """retuns a list of target objects from the inventory"""
+    """returns a list of target objects from the inventory.
```

Comment:
Just nitpicking - to keep comments consistent, for one-liners we close the `"""` on the same line.

Reply:
@adrianchifor thanks for the review! Will be fixing it shortly